src/TaskManager/Plug-ins/Argo/ExitHookTemplate.cs

```diff
@@ -80,7 +80,8 @@ public Template2 GenerateMessageTemplate(S3Artifact2 artifact)
                 TaskId = _taskDispatchEvent.TaskId,
                 ExecutionId = _taskDispatchEvent.ExecutionId,
                 CorrelationId = _taskDispatchEvent.CorrelationId,
-                Identity = "{{workflow.name}}"
+                Identity = "{{workflow.name}}",
+                Outputs = _taskDispatchEvent.Outputs ?? new List<Messaging.Common.Storage>()
```

Comment:
Was just wondering if this is needed given that WM was the one generating this and should have this in the database?

Reply:
It probably isn't needed here as it is passed through, however there may be a time we need to manipulate it in the task executer